src/main/java/gregtech/common/covers/CoverEnderFluidLink.java

```diff
@@ -274,4 +274,8 @@ private void setPrivate(boolean isPrivate) {
         this.isPrivate = isPrivate;
         updateTankLink();
     }
+
```

Comment:
this method is already defined above

Reply:
It wasn't when I started. Glad I wasn't the only one who noticed it was missing.